addons/website_slides/static/tests/tours/slides_course_reviews_comment.js

```diff
@@ -0,0 +1,48 @@
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add("course_reviews_comment", {
+    url: "/slides",
+    steps: () => [
+        {
+            trigger: "a:contains(Basics of Gardening - Test)",
+            run: "click",
+        },
+        {
+            trigger: "a[id=review-tab]",
+            run: "click",
+        },
+        {
+            trigger: "#chatterRoot:shadow .o-mail-Message button:contains('comment')",
+            run: "click",
+        },
+        {
+            trigger: "#chatterRoot:shadow .o-mail-Message .o-mail-Composer-input",
+            run: "edit Putting a comment...",
+        },
+        // When the comment box is closed, the content of the composer is preserved
+        {
+            trigger: "#chatterRoot:shadow .o-mail-Message button:contains('comment')",
+            run: "click",
+        },
+        {
+            trigger: "#chatterRoot:shadow .o-mail-Message:not(:has(.o-mail-Composer))",
+        },
```

Comment:
Should use `.o-mail-Composer-input` in negative part, so the negative assertion matches the positive ones.

This is less of an issue because `.o-mail-Composer-input` implies `.o-mail-Composer` with our BEM convention, but would try to keep the most idiomatic way to trust negative assertions.